index.d.ts

```diff
@@ -279,11 +279,27 @@ declare namespace execa {
 		isCanceled: boolean;
 	}
 
+	interface killOptions {
+		/**
+		If the first signal does not terminate the child after a specified timeout, a `SIGKILL` signal will be sent to the process.
+		*/
+		forceKill?: boolean;
```

Comment:
You should add `@default` for the two options.

Reply:
Done: 99a6aaf7fc8efff0c4c85164ed89b54d4a5f3388